src/setup.py

```diff
@@ -116,10 +116,10 @@ def run(self):
         'vobject==0.9.*',  # 2017-06, http://eventable.github.io/vobject/ look for "release"
         'whitenoise==4.1.*',  # http://whitenoise.evans.io/en/stable/changelog.html
         'zxcvbn==4.4.*',  # Nothing? https://github.com/dwolfhub/zxcvbn-python/issues/38
+        'beautifulsoup4',
```

Comment:
Dependencies are currently ordered alphabetically, pinned to a fixed version, and link their changelog for ease of upgrades. Could you please move bs4 up, pin its version and add a link?